stylesheets/button/button.css

```diff
@@ -0,0 +1,179 @@
+
+.button-types-container
+{
+    display:flex;
+    flex-wrap: wrap;
+    max-width:80vw;
+    padding: 0.6rem;
+    gap:2.5rem;
+    flex-direction: row;
+    align-items: center;
+    justify-content: space-between;
+    flex:1;
+}
```

Comment:
I would suggest that you should use a separate style sheet for documentation so, CSS for this container can be placed there.

Reply:
okay  i will do that